<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gb_humidity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>humidtity</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="le_humidity">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="wdg_humidity" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gb_temperature">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>261</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>temperature</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="le_temperature">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>93</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="wdg_temperature" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gb_waterlevel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>water level</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLineEdit" name="le_waterlevel">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>93</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="wdg_waterlevel" native="true"/>
    </item>
    <item row="1" column="2">
     <widget class="QProgressBar" name="pbar_waterlevel">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>75</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gb_webcam">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>831</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>WebCam</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="lb_webcam">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
